presto-parquet/src/main/java/com/facebook/presto/parquet/cache/CachingParquetMetadataSource.java

```diff
@@ -38,14 +38,24 @@ public CachingParquetMetadataSource(Cache<ParquetDataSourceId, ParquetFileMetada
     }
 
     @Override
-    public ParquetFileMetadata getParquetMetadata(ParquetDataSource parquetDataSource, long fileSize, boolean cacheable)
+    public ParquetFileMetadata getParquetMetadata(
+            ParquetDataSource parquetDataSource,
+            long fileSize,
+            boolean cacheable,
+            long modificationTime)
             throws IOException
     {
         try {
             if (cacheable) {
-                return cache.get(parquetDataSource.getId(), () -> delegate.getParquetMetadata(parquetDataSource, fileSize, cacheable));
+                ParquetFileMetadata fileMetadataCache = cache.get(parquetDataSource.getId(), () -> delegate.getParquetMetadata(parquetDataSource, fileSize, cacheable, modificationTime));
```

Comment:
nit: split and format this long line? your call.

Reply:
Sure, i will split it to another line, thanks.